docs/mdbook/configuration/remotes.md

```diff
@@ -14,4 +14,4 @@ The configuration from `remotes` will be merged to the local config using the fo
 1. Remote configs (`remotes`)
 1. Local overrides (`lefthook-local.yml`)
 
-This priority may be changed in the future. For convenience, if you use `remotes`, please don't configure any hooks.
+This priority may be changed in the future. For convenience, try not to have dependencies between your jobs in lefthook.yml config. Remote hooks should be considered as something like a "standalone package of hooks" rather than "plugin with settings extension".
```

Comment:
@scop WDYT, does it look better?

Reply:
I see. Well, "convenience" is indeed a bad word here, better say robustness and simplicity.

Dependencies between jobs means when you want to have some local jobs with the same names, so that remotes can override them.

And the same thing for "package-like" and "plugin-like" approach. That's actually my intention to provide a best practice to not try having remote jobs for complex and unintuitive flows.